ext/st/README.md

```diff
@@ -1,20 +1,25 @@
 
-- [CubeF0 v1.6.0 (v2.3.0) @ 27-May-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef0.html)
-- [CubeF1 v1.4.0 (v4.1.0) @ 29-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef1.html)
-- [CubeF3 v1.5.0 (v2.3.0) @ 29-April-2015](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef3.html)
-- [CubeF4 v1.12.0 (v2.5.0) @ 22-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef4.html)
-- [CubeF7 v1.4.0 (v1.1.0) @ 22-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef7.html)
+Here is the list of the current Cube version, device header version in braces and release date of the device headers:
+
+- [CubeF0 v1.7.0 (v2.3.1) released 04-November-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef0.html)
+- [CubeF1 v1.4.0 (v4.1.0) released 29-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef1.html)
+- [CubeF2 v1.4.0 (v2.1.2) released 29-June-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef2.html)
+- [CubeF3 v1.7.0 (v2.3.1) released 16-December-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef3.html)
+- [CubeF4 v1.14.0 (v2.5.0) released 22-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef4.html)
+- [CubeF7 v1.6.0 (v1.1.0) released 22-April-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubef7.html)
+- [CubeL0 v1.8.0 (v1.7.1) released 25-November-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubel0.html)
+- [CubeL1 v1.6.0 (v2.2.0) released 01-July-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubel1.html)
+- [CubeL4 v1.6.0 (v1.8.0) released 28-October-2016](http://www.st.com/content/st_com/en/products/embedded-software/mcus-embedded-software/stm32-embedded-software/stm32cube-embedded-software/stm32cubel4.html)
```

Comment:
Hm, and also (v1.8) is completely wrong. Was probably too worked up by all the trailing whitespace to do this properly.